### PR TITLE
Fixed (and added) tests for trimmer set

### PR DESCRIPTION
**Pull request by mudbri**
All tests run smoothly with the build.sh scripts for the trimmer applications. To test any example, go to the example's directory and run "make test".

### CODE REVIEW

File: examples/trimmer/airtun/test.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+echo "Test airtun"
+sudo ./airtun-ng_occamized_stripped > t_out 2>&1
```
Comment (caballa) [inline examples/trimmer/airtun/test.sh]: Why do we need `sudo` ?

File: examples/trimmer/curl/install_dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+sudo apt-get install -y libssl-dev
+
+mkdir -p build
+cd build
+# needs SSL 1.0.2g
+wget https://www.openssl.org/source/openssl-1.0.2g.tar.gz
+tar xvfz openssl-1.0.2g.tar.gz
+cd openssl-1.0.2g
+./config --prefix=/usr/local shared
+sudo make install
```
Comment (caballa) [inline examples/trimmer/curl/install_dependencies.sh]: Why `sudo` here?

File: examples/trimmer/dnsproxy/Makefile [hunk 3]
```diff
@@ -37,7 +37,7 @@ ${OUT_DIR}:
 	mkdir -p ${OUT_DIR}
 
 test:
-	python3 test_dns.py
+	sudo python3 test_dns.py
```
Comment (caballa) [inline examples/trimmer/dnsproxy/Makefile:40]: why sudo here?

File: examples/trimmer/totd/Makefile [hunk 4]
```diff
@@ -33,7 +33,7 @@ ipv4: ${OUT_DIR} ${TOTD_BITCODE}
 	python ${TRIMMER_HOME}/tool/trimmer.py ${TOTD_MANIFEST} ./ no-optimization 2> ${LOG_FILE}
 
 test:
-	python3 test_totd.py
+	sudo python3 test_totd.py
```
Comment (caballa) [inline examples/trimmer/totd/Makefile]: Again why sudo here?

**Review by caballa — CHANGES_REQUESTED** [review]
Having `make test` is great. Thanks!

My only questions are about the use of sudo.

File: examples/trimmer/airtun/test.sh (hunk 1 above)
Comment (mudbri) [reply examples/trimmer/airtun/test.sh]: Running airtun without sudo gives the following error: socket(PF_PACKET) failed: Operation not permitted
This program requires root privileges.

File: examples/trimmer/curl/install_dependencies.sh (hunk 2 above)
Comment (mudbri) [reply examples/trimmer/curl/install_dependencies.sh]: Removed this sudo

File: examples/trimmer/dnsproxy/Makefile (hunk 3 above)
Comment (mudbri) [reply examples/trimmer/dnsproxy/Makefile:40]: Running airtun without sudo gives the following error: unable to bind socket: Permission denied

File: examples/trimmer/totd/Makefile (hunk 4 above)
Comment (mudbri) [reply examples/trimmer/totd/Makefile]: Removed sudo from here

**Comment by caballa** [comment]
Ready to merge?

**Comment by mudbri** [comment]
> Ready to merge?

Yes

**Pull request merged**